src/Queue/Failed/MongoFailedJobProvider.php

```diff
@@ -75,12 +76,43 @@ public function find($id)
     /**
      * Delete a single failed job from storage.
      *
-     * @param mixed $id
+     * @param string $id
      *
      * @return bool
      */
     public function forget($id)
     {
         return $this->getTable()->where('_id', $id)->delete() > 0;
     }
+
+    /**
+     * Get the IDs of all of the failed jobs.
+     *
+     * @param  string|null $queue
+     *
+     * @return array
+     */
+    public function ids($queue = null)
+    {
+        return $this->getTable()
```

Comment:
Does `getTable()` need different typing information for static analysis?

I see [`DatabaseFailedJobProvider::getTable()`](https://laravel.com/api/11.x/Illuminate/Queue/Failed/DatabaseFailedJobProvider.html#method_getTable) returns [Illuminate\Database\Query\Builder](https://laravel.com/api/11.x/Illuminate/Database/Query/Builder.html). Is that a parent class shared by the MongoDB library's builder, and thus still statisfied?